exception_reports/storages.py

```diff
@@ -0,0 +1,58 @@
+import io
+import logging
+import os
+import os.path
+
+import tinys3
+
+logger = logging.getLogger(__name__)
+
+
+class ErrorStorage(object):
+
+    def write(self, filename, data):
+        pass
+
+
+class LocalErrorStorage(ErrorStorage):
+
+    def __init__(self, output_path='/tmp/python-error-reports/', prefix=''):
+        self.output_path = output_path
+        self.prefix = prefix
+
+    def write(self, filename, data):
+        output_path = str(self.output_path)
+        if not output_path[-1] == '/':
+            output_path += '/'
+        filepath = os.path.abspath(output_path + self.prefix + filename)
```

Comment:
super minor: os.path.join or PathLib so you don't haver to think about trailing slash in path